### DISCUSSION TITLE
Can deepgram automatically expand contractions like "you're" and "I've" to "you are" and "I have"?

### DISCUSSION THREAD

**Discussion by Mufasa** [deepgram/community#202] [General help] [answered]
I am a newbie and am currently evaluating deepgram for a dictation app that I am writing. Is there any option to automatically expand contractions like "you're" and "I've" to "you are" and "I have"?

**Comment by jkroll-deepgram** [comment]
Currently the best way to accomplish this is through the [Find and Replace](https://developers.deepgram.com/docs/find-and-replace) feature. You can send your request to a URL with the `replace` parameter such as `https://api.deepgram.com/v1/listen?replace=you're:you%20are&replace=i've&I%20have` . Note that the word to find should be lowercase, such as `i've`, while the word to replace can include uppercase, but needs to encode spaces, such as `I%20have`.

**Comment by jkroll-deepgram** [comment, marked as answer]
Another option is to use another module for post-processing of Deepgram transcripts. For JavaScript, you can check out [nlp-expand-contractions](https://github.com/stdlib-js/nlp-expand-contractions), or for Python there is [contractions](https://pypi.org/project/contractions/).
